tmt/steps/report/reportportal.py

```diff
@@ -203,8 +234,7 @@ def get_headers(self) -> dict[str, str]:
                 "Content-Type": "application/json"}
 
     def get_url(self) -> str:
-        api_version = os.getenv(
-            'TMT_PLUGIN_REPORT_REPORTPORTAL_API_VERSION') or self.DEFAULT_API_VERSION
+        api_version = self.data.api_version
```

Comment:
You can inject `self.data.version` into the string directly, is there a need for `api_version` variable?